docs/acme-divergences.md

```diff
@@ -0,0 +1,82 @@
+
+Current draft: [`draft-ietf-acme-acme-03`](https://tools.ietf.org/html/draft-ietf-acme-acme-03).
+
+## [Section 5.5.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-5.5)
+
+Boulder does not provide a `Retry-After` header when a user hits a rate-limit, nor
+does it provide `Link` headers to further documentation on rate-limiting.
+
+## [Section 5.6.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-5.6)
+
+Boulder doesn't return errors under the `urn:ietf:params:acme:error:` namespace
+but instead uses the `urn:acme:error:` namespace from [draft-ietf-acme-01 Section 5.4](https://tools.ietf.org/html/draft-ietf-acme-acme-01#section-5.4).
+
+Boulder also uses `invalidEmail` in place of the error `invalidContact` defined in
+ACME and doesn't implement the `caa` and `dnssec` errors.
+
+## [Section 6.1.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.1)
+
+Boulder does not implement the `new-application` or `key-change` resources. Instead
+of `new-application` Boulder implements the `new-cert` resource that is defined in
+[draft-ietf-acme-02 Section 6.5](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.5).
+
+## [Section 6.1.1.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.1.1)
+
+Boulder does not implement the `meta` field returned by the `directory` endpoint.
+
+## [Section 6.1.2.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.1.2)
+
+Boulder does not implement the `status`, `applications`, or `certificates` fields
+in the registration object (nor the endpoints the latter two link to).
+
+## [Section 6.1.3.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.1.3)
+
+Boulder does not implement applications, instead it implements the `new-cert` flow
+from [draft-ietf-acme-02 Section 6.5](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.5).
+
+## [Section 6.1.4.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.1.4)
+
+Boulder does not implement the `scope` field in authorization objects.
+
+## [Section 6.2.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.2)
+
+Boulder does not allow `tel` URIs in the registrations `contact` list.
+
+## [Section 6.2.1.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.2.1)
+
+Boulder does not implement key roll-over.
+
+## [Section 6.2.2.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.2.2)
+
+Boulder does not implement account deactivation.
+
+## [Section 6.3.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.3)
+
+Boulder does not implement applications, instead it implements the `new-cert` flow
+from [draft-ietf-acme-02 Section 6.5](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.5).
+
+## [Section 6.5.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.5)
+
+Boulder does not implement the `reason` field for the `revoke-cert` endpoint, unspecified (0)
```

Comment:
Nit: can you use backticks or something for the literal string "unspecified"? It feels like a "computer word" :computer: :books: for lack of a better description :-D